Documentation/CodeSnippets/FlexForms/FeLogin.rst.txt

```diff
@@ -0,0 +1,310 @@
+.. Generated by https://github.com/TYPO3-Documentation/t3docs-codesnippets 
```

Comment:
The file uses a mix of spaces and tabs. So the output in documentation uses 8 spaces for indentation which wastes lot of space.

Also, I find this code example way to long for a code snippet in the docs (many repetitions). Maybe stick to the relevant parts of a FlexForm configuration file and just link to the file on GitHub for further reference.

Reply:
So, I don't like this example either. The whole section could do with an overhaul. But I feel like it is out of context for this PR. Can be replaced with an easier example in a follow up if still needed?

At least this code snippet is now generated instead depending on being copied and adjusted

Reply:
Ok, so I used the more basic example from styleguide now and just linked to the github